internal/server/middleware_test.go

```diff
@@ -59,30 +59,6 @@ func issueToken(t *testing.T, db data.GormTxn, identityName string, sessionDurat
 	return body
 }
 
-func TestRequestTimeoutError(t *testing.T) {
```

Comment:
I removed this test because the function it was testing was removed, and the API timeout behaviour is tested by `TestRequestTimeout` in `routes_test.go`.